src/styles/globalStyles.ts

```diff
@@ -37,15 +38,15 @@ const globalStyles = css`
     --h4: calc(var(--h5) / var(--text-scale-ratio)); // 25
     --h5: calc(1em / var(--text-scale-ratio)); // 20
     --small-text: calc(1em * var(--text-scale-ratio)); // 12.8
-    @media screen and (max-width: 1024px) {
+    @media screen and (max-width: 1023px) {
       --text-scale-ratio: 0.85;
     }
-    @media screen and (max-width: 768px) {
+    @media screen and (max-width: 767px) {
       --text-scale-ratio: 0.9;
     }
 
     /* Spacing (fibonacci) */
-    --space-unit: 1em;
+    --space-unit: 1rem;
```

Comment:
Why was this changed, last time I checked the guild had chosen `em` after ad nauseam discussion.  @nobrayner @merodiro

Reply:
I believe this comes down to a difference in what these values are intended to be used for, and how people are perceiving they should be used for.

For example, an `em` is probably not your best choice if you are going to use it for general "pad this, margin that" - which is what seems people are perceiving they can use this for.

This whole "how is CSS handled" discussion I think needs to be "run through the mob" again, methinks. Specifically, we need to get an agreement of what/how the system will work, document it, and then from there it can be more easily enforced - or changes discussed if there is a valid case for changing it.

Reply:
As I mentioned in discord, separate flexible and fixed spacing scales would be better than forcing it all to be rem-based. `--space-1` being `em` and `--space-fixed-1` being `rem`

Reply:
anywho, i need something so space with, and if I change this back to relative, the I will need to use static numbers in places where this was implemented, otherwise the design breaks, so suggestions are welcome

Reply:
> it might sound dumb, but if I'm writing `--space-fixed-1` ten times more than `--space-1` then i'd rather it'll be the other way around: `--space-1` and `--space-relative-1`

This just sounds like personal preference, about something already decided.

Reply:
that is entirely correct

Reply:
Resolving this, as it can be further discussed in the CSS discussion we have opened.

https://github.com/OpenSourceRaidGuild/website/discussions/79